forch/forchestrator.py

```diff
@@ -96,6 +99,9 @@ def initialize(self):
         self._process_static_device_behavior()
         if self._faucetizer:
             faucetizer.write_behavioral_config(self._faucetizer, self._behavioral_config_file)
+            faucet_config_handler = FaucetConfigFileHandler(
+                self._faucetizer, self._structural_config_file, self._behavioral_config_file)
+            self._file_observer.schedule(faucet_config_handler, os.getenv('FAUCET_CONFIG_DIR'))
```

Comment:
I would push the Observer down into file_handlers -- there's no good reason to expose that part of the abstraction here.

Reply:
Created a FileWatcher class in file_watchers.py